tests/unit/AuthenticateUserTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace unit;
+
+use PHPUnit\Framework\TestCase;
+use Pusher\Pusher;
+use Pusher\PusherException;
+
+class AuthenticateUserTest extends TestCase
+{
+    /**
+     * @var Pusher
+     */
+    private $pusher;
+
+    protected function setUp(): void
+    {
+        $this->pusher = new Pusher('thisisaauthkey', 'thisisasecret', 1, []);
+    }
+
+    public function testObjectConstruct(): void
+    {
+        $this->assertNotNull($this->pusher, 'Created new \Pusher\Pusher object');
```

Comment:
nit why not just `Pusher`?

Reply:
That is a very good question. 🤔

Reply:
Ah, this is meant to mean the `\Pusher\Pusher` class not just Pusher. `\` is the character to separate namespaces.
Haha - your question threw me off.